tests/ArmTemplates.Tests/Creator/TemplateCreatorTests/TagTemplateCreatorTests.cs

```diff
@@ -54,20 +53,17 @@ CreatorParameters GenerateCreatorParameters(List<string> tagNames = null, List<s
             return creatorConfig;
         }
 
-        [Fact]
-        public void CreateTagTemplate_ShouldCreateTemplateFromCreatorConfig_GivenApiTagsAndConfigTags()
+        [Theory]
+        [InlineData("tag 1,tag2,api tag 3,api tag2")]
+        [InlineData("tag 1, tag2, api tag 3, api tag2")]
+        public void CreateTagTemplate_ShouldCreateTemplateFromCreatorConfig_GivenApiTagsAndConfigTags(string apiTagNames)
         {
             var tagTemplateCreator = new TagTemplateCreator(new TemplateBuilder());
             
             var tagNames = new List<string>()
```

Comment:
Thank you for usage and test improvements. 
Since we have changed the apiTagNames generation, keep consistency in tests could we change tagNames to use the same approach ?

Reply:
Well, actually I haven't adjusted the `tagNames` of the generator method as both, the config file and the parsed `CreatorParameters` use a `List` for the "global" tags, while the `ApiConfig` uses a `string` for the local API tags.

Using this approach we simulate a real config input. What do you think?

Reply:
Makes sense, got it.
No updates needed, Thank you!